README.md

```diff
@@ -1,2 +1,37 @@
 # Hello-Microverse
-Microverse project 
+Microverse project
+
+### Built With
+Html<br />
+CSS
+
+### Getting Started 
+To get a local copy up and running follow these simple example steps
+<ul>
+   <li> Prerequisite:  
+        <ol>
+            <li>vscode editor</li> 
+            <li>a web browser(chrome, etc) </li>
+        </ol>
+   </li>
+   <li> <a href = "https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/cloning-and-forking-repositories-from-github-desktop">Clone </a> the repository to your local machine </>
+   <li>Open index.html file with chrome</li>
+
+</ul>
+
+### Author
+<a href="https://github.com/kusiLaw">Github</a> \
+<a href="https://twitter.com/kusilaw">Twitter</a> \
+<a href="https://www.linkedin.com/in/lawrence-kusi-55a662104">Linkedin</a>
+
+
+### Contributing
+Contributions, issues, and feature requests are welcome!
+
+
+### Acknowledgments
+Thanks to Microverse
+
+### License
+This Project is <a href ="https://opensource.org/licenses/MIT">MIT </a> licensed
+
```

Comment:
- [OPTIONAL] I think it would be better if you used the MIT.md file provided by Microverse [this template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) and don't forget to customize it 👌.